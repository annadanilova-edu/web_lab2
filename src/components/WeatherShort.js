import React from 'react';

export default class WeatherShort extends React.Component{
    render(){
        return(
            <div>
                <h2>CityName</h2>
            </div>
        )
    }
}