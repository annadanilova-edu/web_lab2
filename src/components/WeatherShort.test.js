import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import configureMockStore from "redux-mock-store"
import {configure, shallow} from "enzyme";
import WeatherShort from './WeatherShort';

configure({ adapter: new Adapter() });

describe('WeatherShort', () => {
    it('renders correctly', () => {
        const mockStore = configureMockStore();
        const store = mockStore({
            city: {
                city: "Shuzenji",
                temp: 28,
                icon: "01d",
                windSpeed: 0.47,
                windDirection: "north",
                clouds: "broken clouds",
                pressure: 1016,
                humidity: 93,
                lat: 35,
                lon: 139
            }
        });
        const wrapper = shallow(<WeatherShort store={store}/>);
        expect(wrapper.firstChild).toMatchSnapshot();
    })
});