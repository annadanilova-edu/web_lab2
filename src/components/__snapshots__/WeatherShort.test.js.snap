// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherShort renders correctly 1`] = `undefined`;
